settings-development.sample.json

```diff
@@ -39,6 +39,17 @@
       "enabled": true,
       "email": "ct-remote-email"
     },
-    "uploadTransport": "ddp"
+    "uploadTransport": "ddp",
+    "chat": {
```

Comment:
README.md should be updated with theses new settings

Reply:
Fixed in f5f303a75392b0a3702ae5c135722dd546d63cb1